net/pfSense-pkg-WireGuard/Makefile

```diff
@@ -0,0 +1,79 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-WireGuard
+PORTVERSION=	0.0.6
+PORREVISION=	3
+CATEGORIES=	net
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+
+MAINTAINER=	rcmcdonald91@gmail.com
+COMMENT=	pfSense package WireGuard
+
+LICENSE=	APACHE20
+
+RUN_DEPENDS=	${LOCALBASE}/bin/bash:shells/bash \
+		wg:net/wireguard-tools \
+		wg-quick:net/wireguard-tools \
+		${KMODDIR}/if_wg.ko:net/wireguard-kmod
+
+NO_BUILD=	yes
+NO_MTREE=	yes
+
+SUB_FILES=	pkg-install pkg-deinstall
+SUB_LIST=	PORTNAME=${PORTNAME}
+
+do-extract:
+	${MKDIR} ${WRKSRC}
+
+do-install:
+	${MKDIR} ${STAGEDIR}/etc/inc/priv
+
+	${MKDIR} ${STAGEDIR}${PREFIX}/pkg
+	${MKDIR} ${STAGEDIR}${PREFIX}/pkg/wireguard
```

Comment:
`${MKDIR}` is `mkdir -p` so when you create `${STAGEDIR}${PREFIX}/pkg/wireguard`, it will automatically create `${STAGEDIR}${PREFIX}/pkg`